charts/generic-govuk-app/templates/transition-config-cron.yaml

```
@@ -0,0 +1,92 @@
+{{- if .Values.transitionConfigEnabled -}}
+{{- $fullName := include "generic-govuk-app.fullname" . }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ $fullName }}-transition-config-cron
+  labels:
+    {{- include "generic-govuk-app.labels" $ | nindent 4 }}
+    app: {{ $fullName }}-transition-config-cron
+    app.kubernetes.io/name: {{ $fullName }}-transition-config-cron
+    app.kubernetes.io/component: transition-config-cron
```

Similar idea here:
```suggestion
    app.kubernetes.io/component: config-cron
```
(and similarly below)